css/src/components/form/form.scss

```diff
@@ -65,3 +66,9 @@ $field-error-color: $danger !default;
 	content: '*';
 	vertical-align: top;
 }
+
+.form-error-container {
```

Comment:
Just musing here, resolve as you please, but part of me thinks that we should implement [banners](https://www.figma.com/file/uVA2amRR71yJZ0GS6RI6zL/%F0%9F%8C%9E-Atlas-Design-Library?node-id=1002%3A7106)

Reply:
It's also worth nothing, this particular class can easily be made with atomics. Do we need the class?
```
color-danger border-color-danger border
```

Reply:
You're absolutely right. Originally, I added this class because we haven't implemented banners yet. Replaced with atomics for now, as I'd rather not hastily try to implement banners, given the timeline.